react/src/js/react/discover/PostsStore.js

```diff
@@ -66,7 +67,7 @@ export class PostsStore extends Reflux.Store {
         RestUtil.sendGET("posts/", {
             page: page
         }).then((response) => {
-            const posts = update(this.state.posts, {
+            const posts = update(this.state.allPosts, {
```

Comment:
What's the use of allPosts here? It doesn't seem to ever be set in the state of this store

Reply:
I'm basically using it to store the data in an array, because when i combine the posts this.state.posts is an object and I can't use update on the object.

Reply:
You're never setting "allPosts" anywhere except in the initial state though, so can't you just iterate through response.data.results directly? That's also an array